### PR TITLE
fix: Fix issue when user specifies a port for feast ui 

### PR DESCRIPTION
**Pull request by adchia**
**What this PR does / why we need it**:
Changes the JS to look for a fixed projects-list.json (aka the default), and now `feast ui` will write to that json file so we can configure the host + port appropriately.

Before, this was hard-coded so users could not actually control the host / port.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sdk/python/feast/ui/public/projects-list.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (achals) [inline sdk/python/feast/ui/public/projects-list.json:1]: Is this file needed at all, if the `feast ui` command is generating it on the fly?

Comment (adchia) [reply sdk/python/feast/ui/public/projects-list.json:1]: I changed this because to generate it on the fly, I need to hard-code a host + port in that initial React component initialization, which wouldn't work properly

**Review by achals — APPROVED** [review]
/lgtm
